.github/workflows/check-catalyst.yaml

```diff
@@ -421,6 +421,10 @@ jobs:
         # macOS requirements.txt
         python3 -m pip install cuda-quantum==0.6.0
         python3 -m pip install oqc-qcaas-client
+        # TODO: remove workaround before merging to main
+        pip install pennylane-lightning==0.38
```

Comment:
@rauletorresc probably need to change that

Reply:
Done. Thanks!